core/src/main/scala/org/apache/spark/sql/sources/StoreStrategy.scala

```diff
@@ -52,7 +54,16 @@ object StoreStrategy extends Strategy {
     case drop: DropTable =>
       ExecutedCommandExec(drop) :: Nil
 
-    case DMLExternalTable(name, storeRelation: LogicalRelation, insertCommand) =>
+    case p: EncoderPlan[_] =>
+      val plan = p.asInstanceOf[EncoderPlan[Any]]
+      EncoderScanExec(plan.rdd.asInstanceOf[RDD[Any]],
+        plan.encoder, plan.isFlat, plan.output) :: Nil
+
+    case logical.InsertIntoTable(l@LogicalRelation(p: PlanInsertableRelation,
```

Comment:
"create as select" will go to ColumnFormatRelation.insert which has been changed to use dataFrame.write itself (https://github.com/SnappyDataInc/snappydata/pull/449/files#diff-f95903beca8b4604e25ee9f70dd382ceL203) so it will come back to this case.